lib/bin/lwm2m_carrier/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,34 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 0.9.0
+**********************
+
+Snapshot release for modem firmware version 1.2.0 and the |NCS| 1.3.0.
+
+This release is intended to let users begin the integration on the At&T device management platform and start the certification process with AT&T.
+
+Certification status
+====================
+
+The library is not certified with any carrier.
+
+Changes
+=======
+
+* Added new APIs to support portfolio objects. A new portfolio object instance can be created using ``lwm2m_carrier_portfolio_instance_create()``.
+  ``lwm2m_carrier_identity_read()`` and ``lwm2m_carrier_identity_write()`` are used to read and write to corresponding Identity resource fields of
+  a given Portfolio object instance.
+* Expanded API with ``certification_mode`` variable that chooses between certification or test servers upon initialization the LwM2M carrier library.
+* Expanded API with ``apn`` variable to set a custom APN upon initialization the LwM2M carrier library.
+* Added Kconfig configuration "LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_PSK". PSK Key is now set independantly of custom URI.
+* PSK format has changed to be more copy paste friendly.
+  * Was byte array, for example ``static const char bootstrap_psk[] = {0x01, 0x02, 0xab, 0xcd, 0xef};``
+  * Is now a null-terminated string that must be composed of hexadecimal numbers. For example ``static const char bootstrap_psk = "0102abcdef";``.
+* Added lwm2m_carrier_event_deferred_t to retrieve the event reason and timeout.
+* Added FOTA errors to LWM2M_CARRIER_EVENT_ERROR event.
+
+
```

Comment:
Same "Known issues and limitations" as before regarding DTLS/TLS sessions that can be run in parallel. This should be maybe mentioned (same as below for the previous versions).